Podfile

```diff
@@ -14,6 +14,9 @@ target 'Buddies' do
   # Login Flow
   pod 'FBSDKLoginKit'
   
+  # UI Elements 
+  pod 'MaterialComponents/Buttons'
```

Comment:
So my big question with adding this Pod is does it actually provide us with anything. By that, I mean that you have to add so much to the button to get it to do everything we want that I'm curious if it's worth it to increase our build times by having another dependency. I feel as if getting the button to look like a hovering circle would only take a few lines of code beyond what's already in the FAB class. If not, then I'm onboard to add the material components, but if it would be really straightforward to just pop a shadow on a button with a solid background, then I'm less onboard. Thoughts?

Reply:
I guess there is also the neat tapping animation :P Would it be possible to reduce the shadow of the button? IMO it looks a little ugly... it just feels a bit chunky compared to the "slim" feel of iOS